karaf/Dockerfile.rhel

```diff
@@ -27,7 +27,6 @@ USER root
 ADD jboss.repo /etc/yum.repos.d/jboss.repo
 RUN yum install -y --disablerepo=\* --enablerepo=jboss maven \
  && rpm -e --nodeps java-1.7.0-openjdk-devel java-1.7.0-openjdk java-1.7.0-openjdk-headless \
```

Comment:
Forcing removal of RPMs isn't great normally as it means other stuff could now be broken. But not much you can do if Maven 3.0.5 has a hard dependency on Java 7. That's a really old version as I'm sure you know. Is that going to be a problem for users?